### ISSUE TITLE
condition(if) on loop show error not defined

### ISSUE THREAD

**Issue by therajuahammad** [haoxins/gulp-file-include#172]
**Hello,**
I use condition if on loop file. then i call this on index file . but show error condition variable isn't defined. 

Please See screenshots for details:
[http://prntscr.com/ojqkxt](http://prntscr.com/ojqkxt)
[http://prntscr.com/ojqkxt](http://prntscr.com/ojqlca)
[http://prntscr.com/ojqkxt](http://prntscr.com/ojqloq)

Thanks.

**Comment by Gaslan** [comment]
Did you put `blog` variable into `context` data?

**Comment by rajuahammadfanz** [comment]
yes i want call blog variable when call loop file on index.html or others file

**Comment by lmitton** [comment]
Passed the name variable with the ```@@include('includes/navbar.html', {"name" : "Test"})```
Within navbar.html it replaces the @@name value and I get Test, however when I try an if statement
```@@if (name === "Test"){<li>Hello</li>}```
I get the following error.
message: 'name is not defined:  (name === "Test")',
  plugin: 'gulp-file-include',
